packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -13,39 +10,13 @@ import {
   treeFilter,
   pathToNode,
   getTextFromNode,
-  buildPredicate,
 } from 'enzyme/build/RSTTraversal';
 import { describeIf } from './_helpers';
 import { REACT013 } from './_helpers/version';
 
 const $ = elementToTree;
 
 describe('RSTTraversal', () => {
-  describe('splitSelector', () => {
-    const fn = splitSelector;
-    it('splits multiple class names', () => {
-      expect(fn('.foo.bar')).to.eql(['.foo', '.bar']);
-      expect(fn('.foo.bar.baz')).to.eql(['.foo', '.bar', '.baz']);
-    });
-
-    it('splits tag names and class names', () => {
-      expect(fn('input.bar')).to.eql(['input', '.bar']);
-      expect(fn('div.bar.baz')).to.eql(['div', '.bar', '.baz']);
-      expect(fn('Foo.bar')).to.eql(['Foo', '.bar']);
-    });
-
-    it('splits tag names and attributes', () => {
-      expect(fn('input[type="text"]')).to.eql(['input', '[type="text"]']);
-      expect(
-        fn('div[title="title"][data-value="foo"]'),
-      ).to.eql(['div', '[title="title"]', '[data-value="foo"]']);
-    });
-
-    it('throws for malformed selectors', () => {
-      expect(() => fn('div[data-name="xyz"')).to.throw(/Enzyme::Selector received what appears to be a malformed string selector/);
```

Comment:
does this no longer throw (somewhere)? it's a malformed selector.

Reply:
It does throw, but since the new `rst-selector-parser` manages all parsing this should be covered by [the unit tests for the package](https://github.com/aweary/rst-selector-parser/tree/master/test). I figure at this point we'd be testing the behavior of a dependency with its own tests.

But I'll add back a test that ensures that we catch the general parsing error and re-throw something more specific. Then I think it's safe to assume invalid selectors will throw, and if the `rst-selector-parser` coverage is missing a case we can add it there.

On that note, I'd be happy to include `rst-selector-parser` as part of this monorepo if we want to.